pkgs/top-level/perl-packages.nix

```
@@ -6257,12 +6257,12 @@ with self; {
 
   DateExtract = buildPerlPackage {
     pname = "Date-Extract";
-    version = "0.06";
+    version = "0.07";
     src = fetchurl {
-      url = "mirror://cpan/authors/id/A/AL/ALEXMV/Date-Extract-0.06.tar.gz";
-      hash = "sha256-vHZY1cUMNSXsDvy1Ujal3i1dT8BvwUf6OSnI8JU82is=";
+      url = "mirror://cpan/authors/id/E/ET/ETHER/Date-Extract-0.07.tar.gz";
+      hash = "sha256-+geIBK3k7uwd4UcuDguwR65i5MjU1QIHAbnlBXfFuPQ=";
     };
-    buildInputs = [ TestMockTime ];
+    buildInputs = [ TestMockTime TestMockTimeHiRes ];
     propagatedBuildInputs = [ DateTimeFormatNatural ];
```

```suggestion
    buildInputs = [ TestMockTimeHiRes ];
    propagatedBuildInputs = [ ClassDataInheritable DateTimeFormatNatural ];
```

.. so we don't get a conflict with d6efde33dc80023d8976c2a4cdeac1440554d141

---

Sorry, this also depends on 30975d969998a6ec618a4d15768a2ac3412592fe which adds `propagatedBuildInputs = [ TestMockTime ];` to `TestMockTimeHiRes`.

You can add back `TestMockTime` as a buildInput here and I'll sort it out when it reaches staging-next.

---

@dasJ Added a PR to this PR here: https://github.com/helsinki-systems/nixpkgs/pull/3